packages/peregrine/lib/talons/CartPage/useCartPage.js

```diff
@@ -18,7 +18,7 @@ export const useCartPage = props => {
 
     const { called, data, loading } = useQuery(getCartDetails, {
         fetchPolicy: 'cache-and-network',
-        // Don't make this call if we don't have a cartId
+        nextFetchPolicy: 'cache-first',
```

Comment:
> but do we need to rethink how we design components to work with the intended behavior of cache-and-network

I think so, but it depends on how we interpret [this comment](https://github.com/apollographql/apollo-client/issues/6760#issuecomment-668188727) and what Ben meant by "whenever there's a cache update that affects this query". 

For example, if a query references an object's properties but a mutation modifies some other properties, is the query refetched? Is it only if there is an overlapping property such as if the mutation modified something the query asked for?

The answers to this may affect a large chunk of our application and how we've written mutations.